gdg_events/android_study_jam.md

```diff
@@ -3,4 +3,39 @@
 
 ![](http://developerstudyjams.com/images/masthead.png)
 
+- Google Developers Study Jams is a free series of global, community-run, in-person study groups. 
+- The first Study Jams series was presented in partnership with Udacity and will follow their Android Fundamentals curriculum. 
+- Google Developer Group facilitators certified in the course lead the Study Jams that meet once weekly for 8 - 9 weeks.
+- Study Jams follow the free version of the course on udacity.
+- Promoted using hashtag `#devstudyjam`
 
+
+
+#####WHAT IS GOOGLE DEVELOPERS STUDY JAMS?
+Google Developers Study Jams is an 8-week series of in-person, community-run study groups to support students taking online Udacity courses on Google developer technologies. These events are run by local Google Developer Groups or other developer meetups all over the world. This season, we are running Study Jams around the Udacity [Developing Android Apps: Android Fundamentals](https://www.udacity.com/course/developing-android-apps--ud853) course.
```

Comment:
I would avoid references to dates. And add a section for Android Study jam, instead of mixing in Android in the general description.